tools/c7n_azure/c7n_azure/resources/storage_container.py

```diff
@@ -62,6 +63,21 @@ def extra_args(cls, parent_resource):
             return {'resource_group_name': parent_resource['resourceGroup'],
                     'account_name': parent_resource['name']}
 
+    def get_resources(self, resource_ids):
+        client = self.get_client()
+        data = [
+            self.get_storage_container(rid, client)
+            for rid in resource_ids
+        ]
+        return self.augment([r.serialize(True) for r in data])
+
+    def get_storage_container(self, resource_id, client):
+        parsed = parse_resource_id(resource_id)
```

Comment:
idle thought, stylistically parse_resource_id usage seems to be very common, was thinking a variant that might help ergonomics of using would be returning a namedtuple. rather calling get('x') for attribute access.